.pre-commit-config.yaml

```diff
@@ -10,9 +10,9 @@ ci:
     skip: []
     submodules: false
 repos:
-    - repo: https://github.com/pre-commit/mirrors-autopep8
-      rev: 898691a
+    - repo: https://github.com/hhatto/autopep8
+      rev: 8b75604
       hooks:
       - id: autopep8
         exclude: _pb2.py$
-        args: ["--ignore", "E251,E241,E221", "-i"]
+        args: ["--ignore", "E251,E241,E221,E402,E265,E275", "-i"]
```

Comment:
Now I'm intrigued.  Why do we need this growing list?

I believe this is the corresponding list: https://pycodestyle.pycqa.org/en/latest/intro.html

Reply:
Should we decide on a list of exclusions. I believe this list was to support our existing style, but I think it should be OK to let it all be rescanned even if it leads to a big diff.

Reply:
not sure; I generated this by trying to run the new autopep on the code. I used `pycodestyle` on the files that were failing that I hadn't changed to find out what it was complaining about.

I added ignores until all the currently existing files passed.

Reply:
Good answer.

Some projects have an autoformat feature where it will correct such errors automatically.